library/src/main/java/org/xmtp/android/library/codecs/GroupChatMemberAddedCodec.kt

```diff
@@ -1,11 +1,7 @@
-package uniffi.xmtp_dh.org.xmtp.android.library.codecs
```

Comment:
These some how got added under the Rust package. Most of these package changes are just a clean up item.